05 OwnProps/src/pods/organization/container.js

```diff
@@ -4,17 +4,45 @@ import { OrganizationComponent } from './component'
 import { loadMembersRequest } from './actions';
 
 
+// Temporary container once we add redux support we will remove it
+export class OrganizationContainer extends Component {
+  constructor(props) {
+    super(props);
 
-const mapStateToProps = (state) => ({
+    this.state = {
+      repos: [],
+    }
+  }
+
+  fetchRepos = () => {
+    this.setState({
+      repos: [
+        {id: 1, name: "my fake repo A"},
+        {id: 2, name: "my fake repo B"},
+      ],
+    });
+  }
+
+  render(){ return (
```

Comment:
strange wrapping?

Reply:
You mean it should be?

```javascript
render() {
  return (
 ...
  ) 
}
```

Reply:
Yeah probably, it's the first time I see `render (){ return (` especially as one component before it's like that.